### PR TITLE
Move authentication info from query string to socket.io message.

### PR DESCRIPTION
**Pull request by jianjunz**
Ref: https://github.com/open-webrtc-toolkit/owt-server-p2p/issues/24

### CODE REVIEW

File: src/samples/p2p/js/sc.websocket.js [hunk 1]
```diff
@@ -71,14 +58,13 @@ function SignalingChannel() {
...
-
-    wsServer.on('server-authenticated', function(data) {
-      console.log('Authentication passed. User ID: ' + data.uid);
-      if (connectPromise) {
-        connectPromise.resolve(data.uid);
-      }
-      connectPromise = null;
+      wsServer.emit('authentication', {token: loginInfo.token}, (data) => {
+        console.log('Authentication passed. User ID: ' + data.uid);
+        if (connectPromise) {
```
Comment (taste1981) [inline src/samples/p2p/js/sc.websocket.js:63]: check status for resolving/reject connectPromise.

Comment (jianjunz) [reply src/samples/p2p/js/sc.websocket.js:63]: `connectPromise` is `null` after it is settled.

Comment (taste1981) [reply src/samples/p2p/js/sc.websocket.js:63]: I would expect this:
wsServer.emit('authentication', {token: loginInfo.token}, (status, data) => {.... }

Comment (jianjunz) [reply src/samples/p2p/js/sc.websocket.js:66]: Done. Parsing `data` before resolving `connectPromise`.

**Review by taste1981 — APPROVED** [review]

**Pull request merged**